client/experimentalclient/backup.go

```diff
@@ -0,0 +1,65 @@
+package experimentalclient
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"path"
+
+	"github.com/coreos/etcd-operator/pkg/backup"
+	"github.com/coreos/etcd-operator/pkg/util/k8sutil"
+)
+
+type Backup interface {
+	// Request requests a backup. Request returns nil, once a backup is made successfully.
+	// Or it returns an error.
+	Request(ctx context.Context) error
+}
+
+type backupClient struct {
+	client *http.Client
+	url    string
+}
+
+func NewBackup(c *http.Client, clusterName string) Backup {
+	return &backupClient{
+		client: c,
+		url:    k8sutil.BackupServiceAddr(clusterName),
+	}
+}
+
+func NewBackupWithURL(c *http.Client, url string) Backup {
+	return &backupClient{
+		client: c,
+		url:    url,
+	}
+}
+
+func (b backupClient) Request(ctx context.Context) error {
+	req, err := http.NewRequest(http.MethodGet, fmt.Sprintf("http://%s/backupnow", path.Join(b.url, backup.APIV1)), nil)
```

Comment:
No. This is not what should happen. backup should not import its client and its client should not import the backup server pkg either. there needs to be a backupAPI pkg. we will do that in another pr.